Formula/pdal.rb

```diff
@@ -28,6 +30,13 @@ def install
       args << "-DWITH_LASZIP=FALSE"
     end
 
+    args << "-DBUILD_PLUGIN_PGPOINTCLOUD=ON"
+    args << "-DBUILD_PLUGIN_GREYHOUND=ON"
+    args << "-DBUILD_PLUGIN_PYTHON=ON"
+    args << "-DBUILD_PLUGIN_PCL=ON"
+    args << "-DBUILD_PLUGIN_ICEBRIDGE=ON"
+    args << "-DBUILD_PLUGIN_SQLITE=ON"
```

Comment:
Since we don't have cases anymore, we can simply pass all arguments to `cmake` directly:

`system "cmake", ".", *std_cmake_args, "-D....", etc`